include/miopen/miopen.h

```diff
@@ -337,8 +337,27 @@ typedef enum
     miopenBFloat16 = 5, /*!< 16-bit binary floating point (8-bit exponent, 7-bit fraction)
                            (Partially supported) */
     miopenDouble = 6,   /*!< 64-bit floating point (Partially supported) */
+    miopenHalfx4 = 7,   /*!<Pack of 4 16-bit floating point values. Suitable for vectorized tensor
+                           formats only.*/
+    miopenHalfx8 = 8,   /*!<Pack of 8 16-bit floating point values. Suitable for vectorized tensor
+                           formats only.*/
```

Comment:
I suggest we remove these and use special layout enum members to specify the packing dimension. That way the data type enum would only specify what the data type is and the layout enum would specify what the layout and the packing dimension is.

Reply:
Thanks JD, I will modify this part.

Reply:
@JehandadKhan Hi, this change has been implemented, would you like to say something about Artem's question?